files/scripts/swss.sh

```diff
@@ -171,22 +180,29 @@ function clean_up_chassis_db_tables()
     return " 0 $lc $asic
 
     # Next, delete SYSTEM_LAG_MEMBER_TABLE entries
-    $SONIC_DB_CLI CHASSIS_APP_DB EVAL "
+    num_lag_mem=`$SONIC_DB_CLI CHASSIS_APP_DB EVAL "
+    local nlm = 0
     local host = string.gsub(ARGV[1], '%-', '%%-')
     local dev = ARGV[2]
     local ps = 'SYSTEM_LAG_MEMBER_TABLE*|' .. host .. '|' .. dev
     local keylist = redis.call('KEYS', 'SYSTEM_LAG_MEMBER_TABLE*')
     for j,key in ipairs(keylist) do
         if string.match(key, ps) ~= nil then
             redis.call('DEL', key)
+            nlm = nlm + 1
         end
     end
-    return " 0 $lc $asic
+    return nlm" 0 $lc $asic`
+
+    debug "Chassis db clean up for ${SERVICE}$DEV. Number of SYSTEM_LAG_MEMBER_TABLE entries deleted: $num_lag_mem"
```

Comment:
add debug statements for the deletion of other entries?

Reply:
Yes. I'll add debug for other table entries.

Reply:
Fixed. Debug logs added for deletion of other tables also.